components/notebook-controller/config/manager/manager.yaml

```diff
@@ -36,6 +36,16 @@ spec:
               configMapKeyRef:
                 name: config
                 key: ISTIO_GATEWAY
+          - name: ISTIO_HOST
+            valueFrom:
+              configMapKeyRef:
+                name: config
+                key: ISTIO_HOST
+          - name: CLUSTER_DOMAIN
+            valueFrom:
+              configMapKeyRef:
+                name: config
+                key: CLUSTER_DOMAIN
```

Comment:
Lets remove this bit or test this with RHOAI controller as in past, it caused trouble.
https://github.com/opendatahub-io/kubeflow/pull/175

Reply:
Thank you for pointing out this. I removed it from the configuration